Sources/GDBRemote/Session.cpp

```diff
@@ -2906,7 +2906,9 @@ void Session::Handle_vCont(ProtocolInterpreter::Handler const &,
           return;
         }
         if (*eptr++ == ':') {
-          if (!action.ptid.parse(eptr, _compatMode)) {
+          // always parse as GDB multi-process if format is p<pid>.<tid> format
```

Comment:
Why parse in multi-process mode?

Reply:
My read of the docs convinced me the `p<pid>.<tid>` syntax is for GDB in multi-process mode. For GDB not in multi-process mode, it would send plain `<tid>` with an implicit pid of "the only process being debugged". Hard to say what it does in practice, but the parse implementation works the same with `kCompatibilityModeGDBMultiprocess ` and `kCompatibilityModeGDB` so passing either one produces the same behavior.